python/doc/theory/meta_modeling/chaos_basis.rst

```diff
@@ -3,132 +3,191 @@
+    \left\langle \pi^{(i)}_j \; , \; \pi^{(i)}_{k} \right\rangle
+    = \Expect{\pi^{(i)}_j(Z_i) \;  \pi^{(i)}_{k}(Z_i)}
+    =  \delta_{j,k}
 
-  The reader is referred to  for details on the selection of suitable
-  families of orthogonal polynomials. It is assumed that the degree of
-  :math:`\pi^{(i)}_{j}` is :math:`j` for :math:`j>0` and
-  :math:`\pi^{(i)}_{0} \equiv 1` (:math:`i=1,\dots,n`). Upon tensorizing
-  the :math:`n` resulting families of univariate polynomials, one gets a
-  set of orthonormal multivariate polynomials
-  :math:`\{\psi_{\idx}, \idx \in \Nset^n\}` defined by:
+for any :math:`j, k \in \Nset`.
+More details orthogonal polynomials are presented in
+:ref:`orthogonal_polynomials`.
+We make the hypothesis that :math:`\pi^{(i)}_{0} = 1` for :math:`i=1,\dots,n_X`.
+It can be proved that the *tensor product* of the :math:`n_X` univariate
+polynomials produces a set of orthonormal multivariate polynomials
+:math:`\{\psi_{\idx}, \; \idx \in \Nset^{n_X}\}` defined by:
 
-  .. math::
+.. math::
     :label: 3.010six
 
-      \psi_{\idx}(\vect{x}) \, \, \equiv \,\, \pi^{(1)}_{\alpha_{1}}(x_{1}) \times \cdots \times \pi^{(n)}_{\alpha_{n}}(x_{n})
+    \psi_{\idx}(\vect{z})
+    = \pi^{(1)}_{\alpha_1}(z_1) \times \cdots \times \pi^{(n_X)}_{\alpha_{n_X}}(z_{n_X})
 
-  where the multi-index notation
-  :math:`\idx \equiv \{\alpha_{1},\dots,\alpha_{M}\}` has been
-  introduced.
+where :math:`\idx = (\alpha_1,\dots,\alpha_{n_x}) \in \Nset^{n_X}` is
+a multi-index representing the marginal polynomial degrees.
 
-**Building of the PC basis – dependent random variables**
+PC basis with dependent random variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-| In case of *dependent* variables, it is possible to build up an
-  orthonormal basis as follows:
+In case of *dependent* variables, it is possible to build up an
+orthonormal basis as follows (see [soizeghanem2004]_):
 
-  .. math::
+.. math::
     :label: 3.010seven
 
-      \psi_{\idx}(\vect{x}) \, \, = \,\,  K(\underline{x}) \;\prod_{i=1}^M \pi^{(i)}_{\alpha_{i}}(x_{i})
+    \psi_{\idx}(\vect{z})
+    = \left( \frac{f_1(z_1) \cdots f_{n_X}(z_{n_X})}{f_{\vect{Z}}(\vect{z})}\right)^{\frac{1}{2}}
+    \prod_{i=1}^{n_X} \pi^{(i)}_{\alpha_{i}}(z_i).
 
-  where :math:`K(\underline{x})` is a function of the copula of
-  :math:`\vect{X}`. Note that such a basis is no longer polynomial. When
-  dealing with independent random variables, one gets
-  :math:`K(\underline{x})=1` and each basis element may be recast as in
-  :eq:`3.010six`. Determining :math:`K(\underline{x})` is usually
-  computationally expensive though, hence an alternative strategy for
-  specific types of input random vectors.
+Notice that such a basis is no longer made of polynomials. When
+dealing with independent random variables, each basis element may be recast as in
+:eq:`3.010six`.
+Computing :math:`\psi_{\idx}(\vect{z})` can be expensive.
+This is the reason why we present below other methods for specific types
```

Comment:
below? What are you referring to?

Reply:
I mean "the next paragraph". So I replace "below" with "now".

Reply:
Fixed at ea9fdfcdb.